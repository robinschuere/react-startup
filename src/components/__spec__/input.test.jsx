import React from 'react';
import renderer from 'react-test-renderer';
import Input from '../input';

test('Input', () => {
  const onHandleChange = (event) => {
    expect(event).toEqual('something');
  };

  const component = renderer.create(
    <Input label="this" value="is alright" type="text" handleChange={onHandleChange} id="input" />,
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
