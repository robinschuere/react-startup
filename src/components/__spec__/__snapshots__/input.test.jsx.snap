// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input 1`] = `
<div
  className="form-group"
>
  <label
    htmlFor="input"
  >
    this
    <input
      className="form-control"
      id="input"
      onChange={[Function]}
      required={true}
      type="text"
      value="is alright"
    />
  </label>
</div>
`;
